Makefile

```diff
@@ -222,3 +222,9 @@ endif
 .PHONY: help
 help:  ## Display this help menu
 	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-20s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+update-attributions:
```

Comment:
This should probably be added as some step to the CI to ensure that it is always up-to-date (as we do for e.g. manifest generation, etc.)

Reply:
Added a `make verify` that runs a few existing items - including updating attributions, then check the git status.

There are some other points we can improve on around that, but thought to do it as a separate PR. This one is getting too convoluted.